test/debugger/test-debugger-repl.js

```diff
@@ -75,3 +75,7 @@ addTest('for (var i in process.env) delete process.env[i]', []);
 addTest('process.env', [
   /\{\}/
 ]);
+
+addTest('arr = [{foo: "bar"}]', [
+  /\[ \{ foo: 'bar' \} \]/
```

Comment:
nit: while it's not likely to change, it might be worthwhile making the whitespace in the regex optional, just in case. Something like `/\[\s*\{\s*foo\:\s*'bar'\s*\}\s*\]/`